src/fidesops/task/graph_task.py

```diff
@@ -432,7 +447,16 @@ def access_results_post_processing(
         # Return filtered rows with non-matched array data removed.
         return output
 
-    @retry(action_type=ActionType.access)
+    def skip_if_disabled(self) -> None:
+        """Skip execution for the given collection if it is attached to a disabled ConnectionConfig."""
+        connection_config: ConnectionConfig = self.connector.configuration
+        if connection_config.disabled:
+            raise CollectionDisabled(
+                f"Skipping collection {self.traversal_node.node.address}. "
+                f"ConnectionConfig {connection_config.key} is disabled.",
+            )
+
+    @retry(action_type=ActionType.access, default_return=[])
```

Comment:
We used to have this default_return parameter, and it was removed, but I'm restoring it now because it's handy for when a collection is skipped, we just return the "default_return".